Newsletters/Yearn Finance Newsletter #44/Yearn Finance Newsletter #44.md

```diff
@@ -1,95 +1,95 @@
-# Yearn Finance Newsletter #44
+# Newsletter #44 de Yearn Finance 
 
 ![](image1.jfif)
 
-Welcome to the 44th edition of the Yearn Finance Newsletter. Our aim with this newsletter is to keep the Yearn and the wider crypto community informed of the latest news, including product launches, governance changes, and ecosystem updates. If you’re interested in learning more about Yearn Finance, follow our official [Twitter](https://twitter.com/iearnfinance) and [Medium](https://medium.com/iearn) accounts.
+Bienvenidos a la edición número 44 de la Newsletter de Yearn Finance. Con esta newsletter nuestra idea es mantener informada a la comunidad de Yearn y de cripto en general con las últimas noticias, incluyendo lanzamiento de productos, cambios de gobernanza y actualizaciones del ecosistema. Si estás interesado en aprender más sobre Yearn Finance, sigue las cuentas oficiales de [Twitter](https://twitter.com/iearnfinance) y [Medium](https://medium.com/iearn). 
 
-## **Summary**
+## **Resumen**
 
-- YIP-63 Approved 
-- Product User Survey
-- Introducing Four New yVaults
-- Yearn Integrated In Defi Saver
-- Don’t Forget About Woofy
-- Vaults at Yearn
-- Ecosystem News
+- Resultados de YIP-63
+- Encuesta al usuario del producto
+- Introducimos 4 nuevas yVaults
+- Yearn es integrado a Defi Saver
+- No olvidemos a Woofy
+- Las vaults de Yearn
+- Noticias del ecosistema
 
-## **YIP-63 Results**
+## **Resultados de YIP-63**
 
 ![](image2.jfif)
 
-The recent vote for YIP-63 has resulted in agreement for the funding of the LeXpunK DAO Yearn has transferred one million DAI to a multisig controlled by Curve Finance, Lido Finance, and LeXpunK DAO.
+La votación reciente con respecto a YIP-63, ha resultado en el acuerdo de aportar fondos al DAO LeXpunK. Yearn ha transferido un millón de DAI al multifirma controlado por Curve Finance, Lido Finance y el DAO LeXpunK.
 
-For context, the LeXpunK DAO is a coalition of DeFi builders and lawyers with the goal of creating a legal defense team far greater and more ethically aligned than those at Silicon Valley venture funds. This is an important initiative that will serve to fight for crypto over the next few years or even decades.
+Como contexto, el DAO LeXpunK, es una coalición de constructores de Defi y abogados con el objetivo de crear un equipo de defensa legal mucho mayor y con una ética mas alineada que los VCs de silicon Valley. Es una iniciativa importante que servirá para luchar por el mundo cripto por los próximos años e incluso las próximas décadas.
```

Comment:
AS-IS: Como contexto, el DAO LeXpunK, es una coalición de constructores de Defi y abogados con el objetivo de crear un equipo de defensa legal mucho mayor y con una ética mas alineada que los VCs de silicon Valley. Es una iniciativa importante que servirá para luchar por el mundo cripto por los próximos años e incluso las próximas décadas.

TO-BE: Como contexto, el DAO LeXpunK, es una coalición de constructores de Defi y abogados con el objetivo de crear un equipo de defensa legal mucho mayor y con una ética más alineada que los VCs de Silicon Valley. Es una iniciativa importante que servirá para luchar por el mundo cripto por los próximos años e incluso las próximas décadas.